Makefile

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020 Red Hat, Inc.
+
+BINDIR ?= output
+
+include build/Configfile
+
+USE_VENDORIZED_BUILD_HARNESS ?=
+
+ifndef USE_VENDORIZED_BUILD_HARNESS
+-include $(shell curl -s -H 'Authorization: token ${GITHUB_TOKEN}' -H 'Accept: application/vnd.github.v4.raw' -L https://api.github.com/repos/open-cluster-management/build-harness-extensions/contents/templates/Makefile.build-harness-bootstrap -o .build-harness-bootstrap; echo .build-harness-bootstrap)
+else
+-include vbh/.build-harness-vendorized
+endif
+
+default::
+	@echo "Build Harness Bootstrapped"
+
+.PHONY: deps
+deps:
+	GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
```

Comment:
We don't need `GO111MODULE=on` anymore.  That was a transition flag for previous go releases.

Reply:
done